### ISSUE TITLE
can't redefine __APOLLO_CLIENT__ error in frontend

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#860]
No idea where this came from. I left my computer running overnight, I think it went to sleep, one of my Firefox tabs was open to lexbox. 2 of them had this error

```
exception.message
    can't redefine non-configurable property "__APOLLO_CLIENT__"
exception.stacktrace
    @https://lexbox.org/project/sena-3:1:14888
    @https://lexbox.org/project/sena-3:1:14990 @https://lexbox.org/project/sena-3:1:14994 
```

[trace](https://ui.honeycomb.io/sil-language-forge/environments/prod/result/6DWf1vNqR5A/trace/iXJaQAB2dzD?fields[]=s_name&fields[]=s_serviceName&source=query&span=cdc6520c6a6c95e7)

there was only a single span coming from the svelte frontend client. I've pulled out all the attributes I think might be helpful for when the trace above goes away.

note:  project code was changed.

**Comment by rmunn** [comment]
Probably related to https://www.github.com/apollographql/apollo-client-devtools/issues/1406.

**Comment by myieye** [comment]
Yeah, I'm guessing you have the [Apollo client devtools extension](https://addons.mozilla.org/en-US/firefox/addon/apollo-developer-tools/) installed?
And `app.error.show_user = true` 😞

That would mean that my attempts to detect errors coming from browser extensions has more holes 😭
Don't worry, I'm not going to work on this for the next 10 months 🤣

**Comment by hahn-kev** [comment]
Yeah I think I do have the Apollo extension installed
